airflow/providers/amazon/aws/hooks/ecs.py

```diff
@@ -168,7 +168,7 @@ def __init__(
         self.log_group = log_group
         self.log_stream_name = log_stream_name
 
-        self.hook = AwsLogsHook(aws_conn_id=aws_conn_id, region_name=region_name)
+        self.logs_hook = AwsLogsHook(aws_conn_id=aws_conn_id, region_name=region_name)
```

Comment:
Is this a readability change? Just want to be sure I’m not missing anything.

Reply:
yes it was for readability. 

I thought about making it a cached property, but this is not an operator, it's a thread to launch, so when we create it it's to run it. I think it makes a lot less sense to make it a cached property in this case.

In operators, the `__init__` is called on DAG parsing, so it makes sense to do as much as possible lazily. Here, not so much.